e2e/app/page/home-page.ts

```diff
@@ -24,19 +26,17 @@ export default class HomePage extends AuthenticatedPage {
   async isLoaded(): Promise<boolean> {
     try {
       await Promise.all([
-        this.waitUntilTitleMatch(PAGE.TITLE),
-        this.waitForTextExists(PAGE.HEADER),
-        Link.forLabel(this.page, LABEL_ALIAS.SEE_ALL_WORKSPACES),
-        this.waitUntilNoSpinner(),
+        waitUntilMatchPageTitle(this._page, PAGE.TITLE, 60000),
+        waitUntilNoSpinner(this._page, 60000)
```

Comment:
We could use an invisible element or flag to indicate the loadedness of the page for these tests, though I guess you want to rely on user-visible behaviors.